sharding-core/sharding-core-common/src/test/java/org/apache/shardingsphere/core/strategy/keygen/SnowflakeShardingKeyGeneratorTest.java

```diff
@@ -76,6 +76,21 @@ public void assertGenerateKeyWithSingleThread() {
         assertThat(actual, is(expected));
     }
     
+    @Test
+    public void assertGenerateKeyWithMaxVibrationOffsetBe1LessThanPowerOf2() {
```

Comment:
What is the mean of the test case name: `assertGenerateKeyWithMaxVibrationOffsetBe1LessThanPowerOf2`?